src/Http/Http.Abstractions/src/ProblemDetails/ProblemDetails.cs

```diff
@@ -64,8 +63,6 @@ public class ProblemDetails
     [JsonExtensionData]
     public IDictionary<string, object?> Extensions
```

Comment:
I think this can go back to a getter auto-property. I only changed it to have a backing field to add the attribute.